README.md

```diff
@@ -554,3 +554,13 @@ It should appear on the map as you refresh your local browser.
 - `ERROR: for X Cannot create container for service X: Invalid bind mount spec "<path>": Invalid volume specification: '<volume spec>'`. If you get this error after running `docker-compose up` on Windows, you should tell `docker-compose` to properly understand Windows paths by setting the environment variable `COMPOSE_CONVERT_WINDOWS_PATHS` to `0` by running `setx COMPOSE_CONVERT_WINDOWS_PATHS 0`. You will also need a recent version of `docker-compose`. We have successfully seen this fix work with [v1.13.0-rc4](https://github.com/docker/toolbox/releases/tag/v1.13.0-rc4). More info here: https://github.com/docker/compose/issues/4274.
 
 - No website found at `http://localhost:8000`: This can happen if you're running Docker in a virtual machine. Find out docker's IP using `docker-machine ip default`, and replace `localhost` by your Docker IP when connecting.
+
+- FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory: You can configure the memory allocation using `NODE_OPTIONS=--max-old-space-size=4096` in the "build-release" script in package.json, example: "export `NODE_OPTIONS=--max-old-space-size=4096` NODE_ENV=production && webpack --bail --mode production". Also update to the latest node version.
+
+- Expected linebreaks to be 'LF' but found 'CRLF': Fixed by adding `"linebreak-style": 0` in "rules" in .estlintrc
```

Comment:
I assume this error popped up after you edited a file? I think I'd prefer to keep the rule there but suggest configuring your editor to use LF for line breaks instead of CRLF (the problem is probably Windows specific as Linux and Mac don't use CRLF by default).

I'm not sure how common this problem is, it's probably something Windows users would generally be aware of as it looks like it'd pop up on every JS project that inherits the AirBnB eslint style :thinking:

Reply:
Also, if we keep this around, maybe we could put it in the "**Windows specific**" subsection as well? (see the comment below)

Reply:
Done